sim/ExternalRepos.mk

```diff
@@ -47,3 +47,8 @@ DPI_DASM_SPIKE_HASH   ?= 8faa928819fb551325e76b463fc0c978e22f5be3
 SVLIB_REPO       ?= https://bitbucket.org/verilab/svlib/src/master/svlib
```

Comment:
Does `ci_check` still pass?

Reply:
I will do a full run today to verify that it still passes

Reply:
Looks like `ci_check` passes

Reply:
Excellent 👍